spec/core/data_structures.md

```diff
@@ -416,28 +416,39 @@ func SumTruncated(bz []byte) []byte {
 
 ## ConsensusParams
 
-| Name      | Type                                | Description                                                                  | Field Number |
-|-----------|-------------------------------------|------------------------------------------------------------------------------|--------------|
-| block     | [BlockParams](#blockparams)         | Parameters limiting the size of a block and time between consecutive blocks. | 1            |
-| evidence  | [EvidenceParams](#evidenceparams)   | Parameters limiting the validity of evidence of byzantine behaviour.         | 2            |
-| validator | [ValidatorParams](#validatorparams) | Parameters limiting the types of public keys validators can use.             | 3            |
-| version   | [BlockParams](#blockparams)         | The ABCI application version.                                                | 4            |
+| Name      | Type                                | Description                                                                               | Field Number |
+|-----------|-------------------------------------|-------------------------------------------------------------------------------------------|--------------|
+| block     | [BlockParams](#blockparams)         | Parameters limiting the size of a block and time between consecutive blocks.              | 1            |
+| evidence  | [EvidenceParams](#evidenceparams)   | Parameters limiting the validity of evidence of byzantine behaviour.                      | 2            |
+| validator | [ValidatorParams](#validatorparams) | Parameters limiting the types of public keys validators can use.                          | 3            |
+| version   | [BlockParams](#blockparams)         | The ABCI application version.                                                             | 4            |
+| synchrony | [SynchronyParams](#synchronyparams) | SynchronyParams influence the validity of block timestamps.                               | 5            |
+| timeout   | [TimeoutParams](#timeoutparams)     | TimeoutParams configure the timings of the steps of the Tendermint consensus algorithm.   | 6            |
+| ABCI      | [ABCIParams](#abciparams)           | ABCIParams configure ABCI functionality specific to the Application Blockchain Interface. | 7            |
+
+
+### HashedParams
+Hashed Params is a subset of Consensus Params, it is amino encoded and hashed into the Header.ConsensusHash.
```

Comment:
Is it actually amino-encoded?

Reply:
https://github.com/tendermint/tendermint/blob/65c0fba564a023fe097ec70a1f652664957e6ce3/types/params.go#L385 

Just checked - it's proto encoded.